LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -352,12 +352,17 @@ typedef NS_ENUM(NSInteger, PasscodeType) {
 
 // MARK: - Methods
 
+#ifndef LTH_IS_APP_EXTENSION
```

Comment:
Does passcode work in share and picker extension after applying this change?

Reply:
| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/53881615/141007144-e581211b-64d0-4da4-9f68-9d233f3da3b0.jpeg" width=320>  | <img src="https://user-images.githubusercontent.com/53881615/141007132-700046e6-dde9-48b8-94d9-349b444f9431.jpeg" width=320>  |
| <video src="https://user-images.githubusercontent.com/53881615/141007180-f53e6bb4-65fc-4fa8-b827-474e71035ad0.MP4" width=320>  | <video src="https://user-images.githubusercontent.com/53881615/141007154-50bf6c25-eb7e-4899-9c7c-788be04ac28b.MP4" width=320>  |

Reply:
Thanks for checking Lancy